modules/plottinggl/src/processors/volumeaxis.cpp

```diff
@@ -161,18 +162,29 @@ const ProcessorInfo VolumeAxis::processorInfo_{
     "Plotting",               // Category
     CodeState::Stable,        // Code state
     "GL, Plotting",           // Tags
-};
+    "Renders an x, y, and z axis next to the input volume."_help};
 
 const ProcessorInfo VolumeAxis::getProcessorInfo() const { return processorInfo_; }
 
 VolumeAxis::VolumeAxis()
     : Processor()
-    , inport_{"volume"}
-    , imageInport_{"imageInport"}
-    , outport_{"outport"}
-    , axisOffset_{"axisOffset", "Axis Offset (%)", 0.1f, 0.0f, 10.0f}
+    , inport_{"volume", "Input volume"_help}
+    , imageInport_{"imageInport", "Background image (optional)"_help}
+    , outport_{"outport",
+               "Output image containing the rendered volume axes and the optional input image"_help}
+    , offsetScaling_{"offsetScaling",
```

Comment:
What would be the reason for choosing one over the other here? If all options are necessary we might want to provide some guidelines in the help.

Reply:
Added the following help
```c++
                     R"(Offset scaling affects tick lengths and offsets of axis captions and labels.
                        + *None* No scaling, offsets and lengths are given in world coordinates.
                        + *Min* Relative scaling based on the shortest extent of the volume. Useful 
                                when visualizing a growing volume.
                        + *Max* Relative scaling based on the longest extent of the volume. Useful 
                                when visualizing a shrinking volume.
                        + *Mean* Relative scaling basd on the mean volume extent.
                        + *Diagonal* Relative scaling based on the diagonal of the volume.
                      )"_unindentHelp,
```
Might consider to either use `mean` or `diagonal`, if we want to reduce the number of options.

Reply:
Would it be possible to choose this automatically based on these guidelines with manual override?

Reply:
I don't think it is worth the effort to try and automate that, just use mean or diagonal as default.